doc/DeveloperGuide.md

```diff
@@ -58,6 +58,7 @@ Priority | As a ... | I want to ... | So that I can...
 `* * *` | new user | see usage instructions | refer to instructions when I forget how to use the App
 `* * *` | user | add a new person |
 `* * *` | user | delete a person | remove entries that I no longer need
+`* * *` | user | clear all persons | start a fresh new address book
```

Comment:
Should this really be a 3 star user story? Would clearing the address book be used so often?